ocode_python/core/orchestrator.py

```
@@ -0,0 +1,540 @@
+                return await self._execute_single_task(
+                    task, tool_registry, side_effect_broker
+                )
+
+        results = await asyncio.gather(
+            *[execute_task_with_semaphore(task) for task in tasks],
+            return_exceptions=True,
+        )
+
+        # Convert exceptions to error results
+        processed_results = []
+        for i, result in enumerate(results):
```

Ruff (0.11.9)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)